tests/functional/tests/deep_sleep_core_lcd/suite_core_lcd.cpp

```diff
@@ -0,0 +1,134 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#include "CoreDMA2D.hpp"
+#include "CoreDSI.hpp"
+#include "CoreFont.hpp"
+#include "CoreGraphics.hpp"
+#include "CoreJPEG.hpp"
+#include "CoreJPEGModeDMA.hpp"
+#include "CoreLCD.hpp"
+#include "CoreLTDC.hpp"
+#include "CoreSDRAM.hpp"
+#include "CoreSTM32Hal.h"
+#include "CoreVideo.hpp"
+#include "tests/config.h"
+#include "tests/utils.h"
+#include "tests/utils_sleep.h"
```

Comment:
There are not all necessary + missing one explicit 
`#include "CoreLCDDriverOTM8009A.hpp"`

```suggestion
#include "CoreDSI.hpp"
#include "CoreLCD.hpp"
#include "CoreLCDDriverOTM8009A.hpp"
#include "CoreLTDC.hpp"
#include "CoreSTM32Hal.h"
#include "tests/config.h"
#include "tests/utils.h"
#include "tests/utils_sleep.h"
```